content/departments/tech-ops/tools/computer-setup.md

```diff
@@ -1,28 +1,27 @@
-## Computer requirements
+# Computer requirements
 
 - All full-time teammates must have a Sourcegraph computer
 - All teammates must work from their Sourcegraph computer
-- All teammates must use their Sourcegraph computer for accessing any company data
-- All teammates should adhear to our related usage and access policies:
-  - [Device useage and privacy policy](../process/team_device_usage_privacy.md)
+- All teammates must only use their Sourcegraph computer for accessing [company data](#company-data)
+- Sourcegraph approves the use of Apple MacOs only (you may work with Windows and Linux devices for work that does not involve company data, such as open source coding)
```

Comment:
open source code is company data but classified as public, not sure if we want to change the wording here.

Possibly : 

Sourcegraph approves the use of Apple MacOs only (you may work with Windows and Linux devices for work that does not involve internal, private or confidential company data, such as open source coding)